ios/NewExpensify/Info.plist

```diff
@@ -85,6 +85,8 @@
 	<string>Your camera roll is used to store chat attachments.</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Your photos are used to create chat attachments.</string>
+	<key>ReactNativeShareMenu_AppGroup</key>
```

Comment:
It seems like we added two different keys to represent the app group for RNSM? Only one of HostAppGroup or ReactNativeShareMenu_AppGroup should be used. (I think the latter is probably better, since we're making it clear what it's for).